app/lib/handlers.dart

```diff
@@ -450,6 +457,28 @@ Future<shelf.Response> apiPackagesHandler(shelf.Request request) async {
   return _jsonResponse(json);
 }
 
+/// Handles requests for /flutter
+Future<shelf.Response> flutterHandler(shelf.Request request) async {
+  final String path = request.requestedUri.path;
+  final String basePath = '/flutter/plugins';
+  if (path == '/flutter' || path == '/flutter/') {
+    return _redirectResponse(basePath);
+  }
+  final int page = _pageFromUrl(request.url);
+  if (path.endsWith('.json')) {
+    return packagesHandlerJson(request, page, true,
+        basePath: basePath, detectedType: BuiltinTypes.flutterPlugin);
+  } else if (request.url.queryParameters['format'] == 'json') {
+    return packagesHandlerJson(request, page, false,
+        basePath: basePath, detectedType: BuiltinTypes.flutterPlugin);
```

Comment:
AFAIK we don't have any users, I've just kept the same pattern we had for /packages. Happy to remove it, especially because the URL spec may change in the near future, and the simpler the better for now.